src/features/embalm/stepContent/components/ArchaeologistList.tsx

```diff
@@ -44,102 +48,142 @@ function ArchaeologistListItem({
   isDialing,
   setIsDialing,
   onClick,
-
 }: ArchaeologistListItemProps) {
   const [isPinging, setIsPinging] = useState(false);
-  const { testDialArchaeologist, pingArchaeologist } = useDialArchaeologists(setIsDialing);
-
-  const rowTextColor = isSelected ? (archaeologist.exception ? '' : 'brand.0') : '';
+  const { testDialArchaeologist } = useDialArchaeologists(setIsDialing);
 
-  return (<Tr
-    background={isSelected ? (archaeologist.exception ? 'errorHighlight' : 'brand.700') : ''}
-    onClick={() => {
-      onClick();
+  return (
```

Comment:
I know you didn't add the `ArchaeologistListItem` component, but since it is being modified heavily here this would be the perfect opportunity to put the component into it's own file.